docs/source/java/install.rst

```diff
@@ -82,85 +94,121 @@ Installing Nightly Packages
     These packages are not official releases. Use them at your own risk.
 
 Arrow nightly builds are posted on the mailing list at `builds@arrow.apache.org`_.
-The artifacts are uploaded to GitHub. For example, for 2022/03/01, they can be found at `Github Nightly`_.
+The artifacts are uploaded to GitHub. For example, for 2022/03/19, they can be found at `Github Nightly`_.
```

Comment:
I don't think we need to change the nightly instructions for Flight. We just need to show using os-maven-plugin above.

Reply:
Ah, I see, thanks.